apps/angular/projection/src/app/component/student-card/student-card.component.ts

```diff
@@ -1,40 +1,55 @@
-import { Component, OnInit } from '@angular/core';
-import { FakeHttpService } from '../../data-access/fake-http.service';
+import { AsyncPipe } from '@angular/common';
+import { Component, OnInit, inject } from '@angular/core';
+import {
+  FakeHttpService,
+  randStudent,
+} from '../../data-access/fake-http.service';
 import { StudentStore } from '../../data-access/student.store';
-import { CardType } from '../../model/card.model';
-import { Student } from '../../model/student.model';
-import { CardComponent } from '../../ui/card/card.component';
+import {
+  CardComponent,
+  CardImageDirective,
+  CardListItemDirective,
+} from '../../ui/card/card.component';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
 
 @Component({
   selector: 'app-student-card',
-  template: `
-    <app-card
-      [list]="students"
-      [type]="cardType"
-      customClass="bg-light-green"></app-card>
-  `,
+
+  templateUrl: 'student-card.component.html',
+
   standalone: true,
   styles: [
     `
-      ::ng-deep .bg-light-green {
+      :host {
         background-color: rgba(0, 250, 0, 0.1);
+        width: max-content;
+        height: max-content;
       }
     `,
   ],
-  imports: [CardComponent],
+  imports: [
+    CardComponent,
+    CardImageDirective,
+    CardListItemDirective,
+    ListItemComponent,
+    AsyncPipe,
+  ],
 })
 export class StudentCardComponent implements OnInit {
-  students: Student[] = [];
-  cardType = CardType.STUDENT;
+  private http = inject(FakeHttpService);
+  private store = inject(StudentStore);
 
-  constructor(
-    private http: FakeHttpService,
-    private store: StudentStore,
-  ) {}
+  public students$ = this.store.students$;
```

Comment:
To go a bit further, you should transform this to signal to benefit and get use to the future of angular